kettle/make_json_test.py

```diff
@@ -35,6 +37,257 @@ def test_buckets(self):
+                'finished': 1595286616,
+                'repo_commit': '5a529aa3a0dd3a050c5302329681e871ef6c162e',
+                'repos': {"kubernetes/kubernetes":"master"},
+            },
+        ),
+        (
+            "Pr_decorated",
+            "gs://kubernetes-jenkins/pr-logs/path",
+            [{'name': "Test1", 'failed': True}],
+            {
+                "timestamp":1595277241,
+                "pull":"93264",
+                "repos":{"kubernetes/kubernetes":"master:5feab0"},
+                "repo-version":"30f64c5b1fc57a3beb1476f9beb29280166954d1",
+            },
+            {
+                "timestamp":1595279434,
+                "passed":True,
+                "result":"SUCCESS",
+                "revision":"5dd9241d43f256984358354d1fec468f274f9ac4"
+            },
+            None,
+            None,
+            {
+                'job': 'pr:pr-logs',
+                'path': 'gs://kubernetes-jenkins/pr-logs/path',
+                'test': [{'name': 'Test1', 'failed': True}],
+                'passed': True,
+                'result': 'SUCCESS',
+                'elapsed': 2193,
+                'tests_run': 1,
+                'tests_failed':1,
+                'started': 1595277241,
+                'finished': 1595279434,
+                'repo_commit': '30f64c5b1fc57a3beb1476f9beb29280166954d1',
+                'repos': {"kubernetes/kubernetes":"master:5feab0"},
+            },
+        ),
+        (
+            "Pr_bootstrap",
+            "gs://kubernetes-jenkins/pr-logs/path",
+            [{'name': "Test1", 'failed': True}],
+            {
+                "node": "0790211c-cacb-11ea-a4b9-4a19d9b965b2",
+                "pull": "master:5a529",
+                "repo-version": "v1.20.0-alpha.0.261+06ea384605f172",
+                "timestamp": 1595278460,
+                "repos": {
+                    "k8s.io/kubernetes": "master:5a529",
+                    "k8s.io/release": "master"
+                },
+                "version": "v1.20.0-alpha.0.261+06ea384605f172"
+            },
+            {
+                "timestamp": 1595282312,
+                "version": "v1.20.0-alpha.0.261+06ea384605f172",
+                "result": "SUCCESS",
+                "passed": True,
+                "job-version": "v1.20.0-alpha.0.261+06ea384605f172",
+            },
+            {
+                "node_os_image": "cos-81-12871-59-0",
+                "infra-commit": "2a9a0f868",
+                "repo": "k8s.io/kubernetes",
+                "master_os_image": "cos-81-12871-59-0",
+            },
+            {
+                "k8s.io/kubernetes": "master:5a529",
+                "k8s.io/release": "master"
+            },
+            {
+                'job': 'pr:pr-logs',
+                'executor': '0790211c-cacb-11ea-a4b9-4a19d9b965b2',
+                'path': 'gs://kubernetes-jenkins/pr-logs/path',
+                'test': [{'name': 'Test1', 'failed': True}],
+                'passed': True,
```

Comment:
I did a double-take here, but I think this is right... it's not like kettle interprets the test results, it just translates what it sees